glib2/test/test-variant.rb

```diff
@@ -24,4 +24,71 @@ class TestGLibVariant < Test::Unit::TestCase
                    [variant.type, variant.value])
     end
   end
+
+  sub_test_case ".parse" do
+    test "value: true" do
+      variant = GLib::Variant.parse("true")
+      assert_equal(true, variant.value)
+    end
+
+    test "value: float" do
+      variant = GLib::Variant.parse(Math::PI.to_s)
+      assert_equal(Math::PI, variant.value)
+    end
+
+    test "value: infinity" do
+      variant = GLib::Variant.parse("inf")
+      assert_equal(Float::INFINITY, variant.value)
+    end
+
+    test "value: int vector" do
+      variant = GLib::Variant.parse("[1, 2, 3]")
+      assert_equal([1, 2, 3], variant.value)
+    end
+
+    test "value: integer, type: UINT16" do
+      variant = GLib::Variant.parse("65535", GLib::VariantType::UINT16)
+      assert_equal(65535, variant.value)
+      assert_equal("q", variant.type.to_s)
```

Comment:
Could you use one assertion for related objects for easy to debug? If we use multiple assertions, the test is stopped when an assertion is failed. Rest assertions aren't executed and the result isn't logged.

```suggestion
      assert_equal([65535, "q"], [variant.value, variant.type.to_s])
```